docs/universal/getting-started.md

```diff
@@ -77,31 +74,44 @@ app.get('*', (req, res) => {
     res.render(join(DIST_FOLDER, APP_NAME, 'index.html'), { req });
 });
 
-// Start up the Node server
-app.listen(PORT, () => {
-  console.log(`Node server listening on http://localhost:${PORT}`);
-});
+// If we're not in the Cloud Functions environment, spin up a Node server
+if (!process.env.FUNCTION_NAME) {
+  const PORT = process.env.PORT || 4000;
+  app.listen(PORT, () => {
+    console.log(`Node server listening on http://localhost:${PORT}`);
+  });
+}
 ```
 
 ## 3. Add a Webpack Config for the Express Server
 
-Create a new file named `webpack.server.config.js` to bundle the express app from previous step. 
+Create a new file named `webpack.server.config.js` to bundle the express app from previous step.
 
 
 ```js
 const path = require('path');
 const webpack = require('webpack');
 
-const APP_NAME = 'YOUR_PROJECT_NAME';
+const APP_NAME = 'YOUR_PROJECT_NAME'; // TODO: replace me!
```

Comment:
I think we need more context around `'YOUR_APP_NAME`. Something comment like "This is the same as the folder name...".

Reply:
Better yet, my mid-term plan here is to read the paths from the `angular.json` file; need to fiddle there. SGTY?